add Zalando Postgres Operator v1.3.0

FxKu opened the pull request:
### Updates to existing Operators

* [x] Is your new CSV pointing to the previous version with the `replaces` property?
* [x] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#bundle-format) defined in the `package.yaml` ?
* [x] Have you tested an update to your Operator when deployed via OLM?
* [x] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#sign-your-work)?

### Operator Description must contain (in order)

1. [x] Description about the managed Application and where to find more information
2. [x] Features and capabilities of your Operator and how to use it
3. [x] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [x] Human readable name and 1-liner description about your Operator
* [x] Valid [category name](https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md#categories)<sup>1</sup>
* [x] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [x] Links to the maintainer, source code and documentation
* [x] Example templates for all Custom Resource Definitions intended to be used
* [x] A quadratic logo

J0zi commented:
@FxKu, please bump version, as 1.2.0 already exists. Then, consider adding "replaces" directive to your CSV, pointing to an older version.

FxKu commented:
@J0zi I've added another `1.3.0` folder and also have the [replaces](https://github.com/operator-framework/community-operators/blob/e01aef1adcc17f3cf6ecca3005bb3830f0d88e6e/upstream-community-operators/postgres-operator/1.3.0/postgres-operator.v1.3.0.clusterserviceversion.yaml#L357) property in place. I've also updated the RBAC for v1.2.0 as sort of a fix, but I can remove changes for v1.2.0 if you want.
